doc/windows/package-manager/package/winget-validation.md

```diff
@@ -0,0 +1,119 @@
+ analysis from [here](https://www.microsoft.com/wdsi/filesubmission).
+
+- The application installs and uninstalls correctly for both administrators and non-administrators.
+
+- The installer supports non-interactive modes.
+
+- All manifest entries are accurate and not misleading.
+
+- The installer comes directly from the publisher\'s website.
+
+Please see [Windows Package Manager policies](.\windows-package-manager-policies.md) for a complete list of the policies.
+
+## Pull request labels
+
+During validation, we apply a series of labels to our pull request to
+communicate progress. Some labels will direct the ISV to take action,
+while others will be directed to the Package Manager developers.
+
+### Status Labels
+
+The following table describes the possible **status labels** you will
+encounter:
+
+| **Label** | **Details** |
+|--------------|-------------|
+| <img width=180/> | <img width=500/> |
+| **Azure-Pipeline-Passed** | The manifest has completed the test pass. It is waiting for approval. If no issues are encountered during the test pass it will automatically be approved. If a test fails, it may be flagged for manual review.|
+| **Blocking-Issue** | This label indicates that the **Pull Request** cannot be approved because there is a blocking issue. You can often tell what the blocking issue is by the included error label as well. |
+| **Needs: Attention** | This label indicates that the **Pull Request** needs to be investigated by the Windows Package Manager development team. This is either due to a test failure that needs manual review, or a comment added to the **Pull Request** by the community. |
+| **Needs: author feedback** | Indicates there is a failure with the submission. We will reassign **Pull Request** back to you. If you do not address the issue within 10 days, the bot will close the **pull request**. **Needs: author feedback** labels are typically added when there was a failure with the Pull Request that should be updated, or if the person reviewing the Pull Request has a question. |
+| **Validation-Completed** | Indicates that the test pass has been completed successfully and your **Pull Request** will be merged.|
+
+### Error Labels
+
+The following table describes the possible **error labels** that will be
+encountered. Not all of the error cases will be assigned to the ISV
+immediately. Some may trigger manual validation.
+
+
+| **Label** | **Details** |
+|--------------|-------------|
+|<img width=200/>|<img width=500/>|
+| **Binary-Validation-Error** | The application included in this **Pull Request** failed to pass the **Installers Scan** test. This test is designed to ensure that the application installs on all environments without warningsFor further details on this error, see [binary validation errors](.\b. inary-validation-errors.md). |
```

Comment:
The link isn't rendering correctly, I think it's a typo or something with the spacing in the path.

Reply:
Grrr.  thanks!!!